components/Sponsors.jsx

```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Heading from "./Heading";
+
+const SPONSORS_CONTAINER = "bg-white flex flex-col items-center";
+const HEADING_CONTAINER = "flex flex-row justify-center mb-10";
+const SPONSORS_LEVEL_CONTAINER = "flex flex-row justify-center items-center my-5";
+const CONTENT_CONTAINER = "my-10 font-medium max-w-md text-center";
+const UNDERLINE_CONTAINER = "mb-10";
+const UNDERLINE_PROPERTIES = {
+  width: 177,
+  height: 10,
+};
+
+const Sponsors = () => {
+  // Multiplier is the value you need to multiply height by to get width
+  const goldSponsorsLevelOne = [
+    { file: "Benevity.png", multiplier: 3.216, height: 90 },
+    { file: "Hunter Hub.png", multiplier: 7.02, height: 70 },
+  ];

```

Comment:
Can we take this out of the component and make a constant as well?

Reply:
nit: redirect to the sponsor's website on click?

Reply:
1. Will do
2. Will do